src/utils/selection.js

```diff
@@ -18,7 +20,7 @@ export function getSlotAtX(rowBox, x, rtl, slots) {
 }
 
 export function pointInBox(box, { x, y }) {
-  return y >= box.top && y <= box.bottom && (x >= box.left && x <= box.right)
+  return y >= box.top && y <= box.bottom && x >= box.left && x <= box.right
```

Comment:
This was done automatically, I assume by the linting config that's set up. I'm happy to revert this change if that's preferable.